addons/web/static/src/views/fields/text/text_field.js

```diff
@@ -100,8 +71,10 @@ export const textField = {
     extractProps: ({ attrs, options }) => ({
         placeholder: attrs.placeholder,
         dynamicPlaceholder: options?.dynamic_placeholder || false,
-        dynamicPlaceholderModelReferenceField: options?.dynamic_placeholder_model_reference_field || "",
+        dynamicPlaceholderModelReferenceField:
+            options?.dynamic_placeholder_model_reference_field || "",
         rowCount: attrs.rows && parseInteger(attrs.rows),
+        lineBreaks: options?.lineBreaks !== undefined ? Boolean(options.lineBreaks) : true,
```

Comment:
why don't you use `archParseBoolean`? It feels that your implementation is wrong. does it work for all usecases? `false`, `False`, `"false"`, `"False"`, `0`, `"0"`, `true`, `True`, `"true", ...?